lib/presets/safe.es6

```diff
@@ -24,6 +24,7 @@ export default {
             { convertShapeToPath: false },
         ],
     },
+    minifyConditionalComments: false,
```

Comment:
Disabled by default since it might break the page. See the `Notice` I added in docs:

https://github.com/posthtml/htmlnano/pull/119/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R556-R574